pkg/controller/infrastructure/infraflow/aliclient/types.go

```diff
@@ -31,21 +31,26 @@ func (f FactoryFunc) NewActor(accessKeyID, secretAccessKey, region string) (Acto
 // VPC is the struct for a vpc object
 type VPC struct {
 	Tags
-	Name      string
-	VpcId     string
-	CidrBlock string
-	Status    *string
+	Name          string
+	VpcId         string
+	CidrBlock     string
+	EnableIpv6    bool
+	Ipv6CidrBlock string
+	Status        *string
 }
 
 // VSwitch is the struct for a vswitch object
 type VSwitch struct {
 	Tags
-	Name      string
-	VSwitchId string
-	VpcId     *string
-	CidrBlock string
-	ZoneId    string
-	Status    *string
+	Name            string
+	VSwitchId       string
+	VpcId           *string
+	CidrBlock       string
+	ZoneId          string
+	EnableIpv6      bool
+	Ipv6CidrkSubnet *int
```

Comment:
What is an `IPv6CidrkSubnet`?

Reply:
It is the subnet identifier.

Reply:
Is the `k` in `IPv6CidrkSubnet` a typo?
If it is just the subnet identifier why not just call it `Ipv6Subnet`?

Reply:
Yes, it's a typo, I will change it.